x/evidence/types/evidence.go

```diff
@@ -84,9 +84,9 @@ func (e Equivocation) GetValidatorPower() int64 {
 // GetTotalPower is a no-op for the Equivocation type.
 func (e Equivocation) GetTotalPower() int64 { return 0 }
 
-// ConvertDuplicateVoteEvidence converts a Tendermint concrete Evidence type to
+// ConvertABCIEvidence converts a Tendermint concrete Evidence type to
 // SDK Evidence using Equivocation as the concrete type.
-func ConvertDuplicateVoteEvidence(dupVote abci.Evidence) exported.Evidence {
+func ConvertABCIEvidence(dupVote abci.Evidence) exported.Evidence {
 	return &Equivocation{
```

Comment:
This type should be renamed to something more distinctive in the sdk? Could be confusing but its a more cosmetic thing

Reply:
I'm happy to change it. Do you have any suggestion for a better name?

Reply:
ABCIEvidence? I don't have a strong preference

Reply:
`ABCIEvidence` would suggest that we are constructing an `ABCIEvidence`, here we are taking `abci.Evidence` and converting to `SDK Evidence`.

Reply:
`FromABCIEvidence`?

Reply:
done

Reply:
The function name was fine imo, the evidence type `Equivocation` is what I meant should be renamed

Reply:
The `Equivocation` is defined in `proto/cosmos/evidence/v1beta1/evidence.proto`. I can rename it. Do you have any suggestion? How about `GenEvidence`? Or `BasicEvidence`?

@marbar3778  - do you prefer to revert: FromABCIEvidence -> ConvertABCIEvidence?

Reply:
BasicEvidence, ABCIEvidence doesn't really matter just need to make it simpler for people to understand. Right now it seems there is one evidence when the sdk is handling two different types of evidence but the same way, for now. 

The x/evidence/spec folder also needs updating.